optimum/commands/export/executorch.py

```diff
@@ -126,6 +126,20 @@ def parse_args_executorch(parser):
         required=False,
         help="Maximum sequence length for the model. If not specified, uses the model's default max_position_embeddings.",
     )
+    required_group.add_argument(
+        "--dtype",
+        type=str,
+        choices=["float32", "float16", "bfloat16"],
+        required=False,
+        help="Data type for model weights. Options: float32, float16, bfloat16. Default: float32. For quantization (int8/int4), use the --qlinear arguments.",
+    )
+    required_group.add_argument(
+        "--device",
+        type=str,
+        choices=["cpu", "cuda", "cuda:0", "cuda:1", "cuda:2", "cuda:3"],
```

Comment:
why 0, 1, 2 etc? that seems like a strange thing

Reply:
> Especially optimum-executorch is really for generating aot artifact and we dont know where it will be deployed

I think this should be fine, I don't think the aoti compiled artifact has this GPU index hardcoded in. I think this is convenient if you have 2 GPUs and the first one is occupied / doesn't have enough memory and we want to export on the second GPU.

Reply:
Sure. I do find it a bit awkward but sure.